packages/dds/tree/src/simple-tree/toFlexSchema.ts

```diff
@@ -254,25 +257,3 @@ export function convertNodeSchema(
 	schemaMap.set(brand(schema.identifier), { original: schema, toFlex });
 	return toFlex;
 }
-
-/**
- * A symbol for storing FlexTreeSchema on TreeNodeSchema.
- * Eagerly set on leaves, and lazily set for other cases.
- */
-export const flexSchemaSymbol: unique symbol = Symbol(`flexSchema`);
```

Comment:
Note for reviewers: These were moved to `schemaCaching.ts` to prevent import cycles, and to remove the need to export the symbol.